### ISSUE TITLE
Need space in between paragraphs

### ISSUE THREAD

**Issue by erictleung** [FreeCodeCamp/FreeCodeCamp#6374]
Challenge [Comparison with the Greater Than Operator](http://www.freecodecamp.com/challenges/comparison-with-the-greater-than-operator) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
function myTest(val) {
  if (val) {  // Change this line
    return "Over 100";
  }
  
  if (val) {  // Change this line
    return "Over 10";
  }

  return "10 or Under";
}

// Change this value to test
myTest(10);

```

Usually, paragraphs are separated with an empty line but here it is not. Currently, both paragraphs are in one string with a `<br>` separating the two paragraphs. For this challenge in the `basic-javascript.json` file, the second paragraph should be on its own string so the spacing is consistent with all the other challenges.

![image](https://cloud.githubusercontent.com/assets/2754821/12506466/f82569a0-c0a3-11e5-8cdc-027d63549b6d.png)

**Comment by erictleung** [comment]
Same goes for this challenge as well: http://www.freecodecamp.com/challenges/comparison-with-the-greater-than-equal-to-operator

**Comment by SaintPeter** [comment]
Humm, don't know why I did that.  Yes, these need a proper line break.

**Comment by kirbyblox** [comment]
Is this how you fork? This is my first time.

**Comment by erictleung** [comment]
@kirbyblox I would consult the [contribution guidelines here](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md). It's got a lot of helpful information. The things I can see that need to be changed are:

- the branch name (it needs to be in the form of `fix/xxx` so like `fix/spaces-in-paragraphs` would be good) and
- also the issue number should not be in the commit title. It should be in the commit body and say something like `Closes #6374`. Here's more information about [closing Github issues using commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
